src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -617,9 +628,16 @@ def __call__(
 class FlaxBertEncoder(nn.Module):
     config: BertConfig
     dtype: jnp.dtype = jnp.float32  # the dtype of the computation
+    gradient_checkpointing: bool = False
+    remat_policy: Callable[..., bool] = (None,)  # the gradient checkpointing policy
```

Comment:
Are there multiple policies? Would one every use another one then the default one? Wondering if allowing this parameter to be customizable might be a bit scary for the user and make the whole functionality less understandable. Think I'd prefer to just use the default here and not allow the user to configure it

Reply:
Thanks for the lengthy explanation! If different policies can work best depending on the circumstances, I'm happy to leave as is :-)